### DISCUSSION TITLE
How to get TTM Operating Income from Income Statement?

### DISCUSSION THREAD

**Discussion by Jonnyrowe** [dpguthrie/yahooquery#131] [General]
This is one of the best api's I have used for yahoo finance yet! No others I've worked with have the TTM included. 

When I run a print statement for the income statement I am not seeing all of the income statements data. In the bottom left hand corner it says 5 rows x 42 columns but there it's only showing about 4 columns. Does anyone know what the problem would be here? I am attempting to get TTM Operating Income for a small project I'm doing. 

Would appreciate some help thanks!

**Comment by dpguthrie** [comment]
Do you have some example code to reproduce the issue?
